ui/app/components/license-banners.js

```diff
@@ -31,15 +31,21 @@ export default class LicenseBanners extends Component {
 
   constructor() {
     super(...arguments);
-    // do not dismiss any banners if the user has updated their version
-    const dismissedBanner = localStorage.getItem(`dismiss-license-banner-${this.currentVersion}`); // returns either warning or expired
+    // reset and show a previously dismissed license banner if:
+    // the version has been updated or the license has been updated (indicated by a change in the expiry date).
+    const dismissedBanner = localStorage.getItem(this.dismissedBannerKey); // returns either warning or expired
```

Comment:
I'm still confused what `dismissedBanner` is? That's why I had suggested renaming the variable to `bannerType` because I read this as the const here can equal either 'warning' or 'expired'

Reply:
Got it. Updated the name.